README.rst

```diff
@@ -197,6 +197,12 @@ transport
     set to "websockets" to send MQTT over WebSockets. Leave at the default of
     "tcp" to use raw TCP.
 
+manual_ack
+    defaults to False, allowing the library to acknowledge messages before
```

Comment:
```suggestion
    defaults to False, allowing the library to acknowledge messages automatically after on_message callback return
```

This behavior changed in release 1.6.0 (https://github.com/eclipse/paho.mqtt.python/commit/9a4941efd758ca4231d4f16a8d3951686e24cb92)